protobuf code generated w/ 3.22.0 broken w/ Quarkus

### Describe the bug

Code that is generated with the new protobuf release 3.22.0 does not work in Quarkus.

The exception looks somewhat like this:
```
Caused by: java.lang.IllegalAccessError: class <some-generated-protobuf-code> tried to access method 'com.google.protobuf.LazyStringArrayList com.google.protobuf.LazyStringArrayList.emptyList()' (<some-generated-protobuf-code> is in unnamed module of loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @4f5c30b1; com.google.protobuf.LazyStringArrayList is in unnamed module of loader io.quarkus.bootstrap.classloading.QuarkusClassLoader @6d4a65c6)
```

protobuf code generated w/ versions < 3.22.0 referenced the `public static` field `com.google.protobuf.LazyStringArrayList#EMPTY`, the static method `emptyList()` is package-protected before 3.22.0.

In 3.22.0 `emptyList()` became `public static` and code generated w/ 3.22.0 references that method.

However, the Quarkus bom enforces protobuf-java to 3.21.9, so code generated w/ 3.22.0 won't be able to access the package-protected method `emptyList()`.

I suspect, a bump of protobuf to 3.22.0 in Quarkus (at least on `main`) should help here.

[protobuf release notes](https://github.com/protocolbuffers/protobuf/releases/tag/v22.0)

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -73,7 +73,7 @@
         <!-- Make sure to check compatibility between these 2 gRPC components before upgrade -->
         <grpc.version>1.53.0</grpc.version> <!-- when updating, verify if com.google.auth should not be updated too -->
         <grpc-jprotoc.version>1.2.1</grpc-jprotoc.version>
-        <protoc.version>3.21.9</protoc.version>
+        <protoc.version>3.22.0</protoc.version>
         <protobuf-java.version>${protoc.version}</protobuf-java.version>
         <proto-google-common-protos.version>2.11.0</proto-google-common-protos.version>
 

